policyengine_us/parameters/gov/states/wa/tax/income/credits/working_families_tax_credit/amount.yaml

```diff
@@ -1,28 +1,33 @@
 description: Washington Working Families Tax Credit maximum amount by number of children.
 metadata:
-  label: WA Working Families Tax Credit maximum amount by number of children
-  name: wa_wftc_max_amount
+  label: Washington Working Families Tax Credit maximum amount by number of children
   type: single_amount
   threshold_unit: person
   amount_unit: currency-USD
   period: year
   reference:
-    - title: "RCW 82.08.0206 Exemptions—Working families—Eligible low-income persons—Penalties. (3)(a)"
+    - title: RCW 82.08.0206 Exemptions—Working families—Eligible low-income persons—Penalties. (3)(a)
       href: https://app.leg.wa.gov/RCW/default.aspx?cite=82.08.0206
+    - title: Tax year 2023 applications now being accepted for the Working Families Tax Credit
+      href: https://dor.wa.gov/about/news-releases/2024/tax-year-2023-applications-now-being-accepted-working-families-tax-credit
 brackets:
   - threshold:
       2022-01-01: 0
     amount:
       2022-01-01: 300
+      2023-01-01: 315
   - threshold:
       2022-01-01: 1
     amount:
       2022-01-01: 600
+      2023-01-01: 625
```

Comment:
does this change with inflation as well?

Reply:
seems no